pkg/operators/clustermanager/controllers/clustermanagercontroller/clustermanager_controller.go

```diff
@@ -679,7 +748,11 @@ func cleanUpHub(ctx context.Context, controllerContext factory.SyncContext,
 	}
 
 	// Remove All Static files
+	// API service should be deleted in 0.10.0
```

Comment:
we should not deploy apiservices any more, right?

Reply:
need a todo here to remove this pieces in the future.

Reply:
done